mixer/adapter/stackdriver/log/log.go

```diff
@@ -226,20 +227,35 @@ func toLabelMap(names []string, variables map[string]interface{}, logEntryTypes
 		v := variables[name]
 		switch vt := v.(type) {
 		case string:
-			out[name] = vt
+			out[name] = stripBadUTF8(vt)
```

Comment:
Does this function work here? https://golang.org/pkg/strings/#ToValidUTF8

Reply:
ha. yes. i didn't know that existed.  that's 1.13 only, though. is 1.2 on 1.13?

Reply:
No 1.2 still uses 1.12. So yeah I think it is fine.